internal/pipeline/runtime/container_test.go

```diff
@@ -0,0 +1,66 @@
+// +build docker
+
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package runtime_test
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/GoogleContainerTools/kpt/internal/pipeline/runtime"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestContainerFn(t *testing.T) {
+	// use sed to test instead of calling real docker
```

Comment:
how do you use `sed`?

Reply:
Updated the test but forgot this comment which was copied from kyaml. Will remove it.